packages/web-app/src/views/questionnaire-check-and-confirm.njk

```diff
@@ -50,7 +69,7 @@
 {% block formContent %}
     <h2 class="govuk-heading-l">Check and confirm</h2>
     {{ govukSummaryList({ rows: sections }) }}
-    {% if reviewOutcome === '3' %}
+    {% if reviewOutcome === 'Complete' %}
```

Comment:
We need to look at why this has been changed as well.

Reply:
This is the value that is being passed from review-questionnaire-submission page. This is how- Incomplete / Complete - it was developed originally. But recent change - not by me - changed this to 1, 2 & 3 similar to they do in Team 1.

Reply:
I agree with @demzzydev, this value shouldn't be changed.

Reply:
If the form is passing `Complete` as a value then that's the value that needs to be changed

Reply:
@shepda I can change it but what is the difference in sending the value as '1' or 'Complete'? Is it for db purpose?

Reply:
Reverted the changes as per PR comments